script/github-actions/get-changed-apps.unit.spec.js

```diff
@@ -311,3 +314,106 @@ describe('getChangedAppsString', () => {
 
   after(() => mockFs.restore());
 });
+
+describe('isContinuousDeploymentEnabled', () => {
+  before(() => {
+    mockFs({
+      'src/applications/app1': {
+        'manifest.json': createManifest('app1'),
+        'some-file.js': '',
+        'other-file.js': '',
+      },
+      'src/applications/app2': {
+        'manifest.json': createManifest('app2'),
+        'some-file.js': '',
+        'other-file.js': '',
+      },
+    });
+  });
+
+  it('should return true when an app has continuous deployment enabled', () => {
+    const config = createChangedAppsConfig([
+      { rootFolder: 'app1', continuousDeployment: true },
+    ]);
+    const changedFiles = ['src/applications/app1/some-file.js'];
+
+    const continuousDeployment = isContinuousDeploymentEnabled(
+      changedFiles,
+      config,
+    );
+    expect(continuousDeployment).to.be.true;
+  });
+
+  it('should return false when an app does not have continuous deployment enabled', () => {
+    const config = createChangedAppsConfig([{ rootFolder: 'app1' }]);
+    const changedFiles = ['src/applications/app1/some-file.js'];
+
+    const continuousDeployment = isContinuousDeploymentEnabled(
+      changedFiles,
+      config,
+    );
+    expect(continuousDeployment).to.be.false;
+  });
+
+  it('should return false when at least one app does not have continuous deployment enabled', () => {
+    const config = createChangedAppsConfig([
+      { rootFolder: 'app1', continuousDeployment: true },
+      { rootFolder: 'app2' },
+    ]);
+    const changedFiles = [
+      'src/applications/app1/some-file.js',
+      'src/applications/app2/some-file.js',
+    ];
+
+    const continuousDeployment = isContinuousDeploymentEnabled(
+      changedFiles,
+      config,
+    );
+    expect(continuousDeployment).to.be.false;
+  });
+
+  it('should return true when all apps have continuous deployment enabled', () => {
+    const config = createChangedAppsConfig([
+      { rootFolder: 'app1', continuousDeployment: true },
+      { rootFolder: 'app2', continuousDeployment: true },
+    ]);
+    const changedFiles = [
+      'src/applications/app1/some-file.js',
+      'src/applications/app2/some-file.js',
+    ];
+
+    const continuousDeployment = isContinuousDeploymentEnabled(
+      changedFiles,
+      config,
+    );
+    expect(continuousDeployment).to.be.true;
+  });
+
+  it('should return false when an app has a value that is not `true` stored in `continuousDeployment`', () => {
```

Comment:
Should we throw an error in this case? It might avoid confusion to immediately flag a poorly-formatted file so they can fix it

Reply:
Good point, even though I'd like to think this is unlikely to happen, that would avoid any confusion if someone did try to store a data type that isn't a boolean. Eventually, depending on how complex the config file gets, we may even want to create a schema for it. That would allow us to enforce things like requiring the `slackGroup` and other stuff